app/views/unattended/report_templates/host_-_statuses.erb

```diff
@@ -22,9 +22,10 @@ model: ReportTemplate
 <%- report_headers 'Name', 'Global' -%>
 <%- machine_readable = input('Output style') == 'Machine-readable' -%>
 <%- load_hosts(search: input('hosts'), includes: :host_statuses).each_record do |host| -%>
-<%-   report_row({
+<%- row_hash = {
         'Name': host.name,
         'Global': machine_readable ? host.global_status : host.global_status_label
```

Comment:
You may say, "But these hash keys look like strings!" Which is what I thought too. But Ruby always makes it a Symbol if you use the more modern Hash syntax. To get true string keys you'd have to use `=>`.

Reply:
AFAIK this has always been the case. Only `=>` leaves the keys untouched, `:` there has always converted it to symbols.

Reply:
I think JavaScript object syntax has ruined my brain 🙈